### DISCUSSION TITLE
Stop point

### DISCUSSION THREAD

**Discussion by leehoangthang** [anitagraser/movingpandas#379] [Q&A]
Hi all,
love this package a lot!
I want to determine a stopping point in this data with a time of greater 10 minutes and a distance of less than 100m, but my program does not work.
[GPS.csv](https://github.com/movingpandas/movingpandas/files/15127377/GPS.csv)

thanks you

**Comment by anitagraser** [comment]
Hi @leehoangthang 
Please provide details about the error and share the code to reproduce the error.
